doc/source/api/index_devapi.rst

```diff
@@ -35,11 +39,165 @@ These classes are used for the processing of data by the ``ObsSurface`` processi
 :class:`~openghg.modules.THAMESBARRIER`
     For processing data from the Thames Barrier measurement sites
 
-* Datasource
+Datasource
+^^^^^^^^^^
 
 The Datasource is the smallest data provider within the OpenGHG topology. A Datasource represents a data provider such as an instrument
```

Comment:
What do you mean by "data provider" here - class that provides data?

Reply:
Yeah, that needs changing really as it's confusing. Maybe it should be "store of data" but I think the whole bit could be rewritten.